### DISCUSSION TITLE
Flatpak release of obs-ndi

### DISCUSSION THREAD

**Discussion by graysham** [DistroAV/DistroAV#773] [Ideas]
Any chance of a flatpak of the plug-in? I’m looking to run obs-ndi on my steamdeck (which is arch based) so that I can use it as a ptz controller for shows.

**Comment by Trouffman** [comment]
This is tracked at : https://github.com/obs-ndi/obs-ndi/issues/724#

For anyone coming from search : This is the way : https://github.com/DistroAV/DistroAV/wiki/1.-Installation#flatpak
